sky/cli.py

```diff
@@ -1699,25 +1716,34 @@ def autostop(
     # NOTE(dev): Keep the docstring consistent between the Python API and CLI.
     """Schedule or cancel an autostop or autodown for cluster(s).
 
-    CLUSTERS are the name (or glob pattern) of the clusters to stop.  If both
+    CLUSTERS are the names (or glob patterns) of the clusters to stop. If both
     CLUSTERS and ``--all`` are supplied, the latter takes precedence.
 
-    If --down is passed, autodown (tear down the cluster; non-restartable) is
-    used, rather than autostop (restartable).
+    If ``--down`` is passed, autodown will be used (tear down the cluster;
```

Comment:
High level, need to explain what autostop is from a top-down apporach:

Top: What is autostop/autodown? (Default behavior)

Down: The details of autostop and its arguments

This will require rearranging the comments.